core-web/libs/template-builder/src/lib/components/template-builder/components/remove-confirm-dialog/remove-confirm-dialog.component.ts

```diff
@@ -41,6 +42,7 @@ export class RemoveConfirmDialogComponent {
     }
 
     openConfirmationDialog(event: Event): void {
+        this.deleteRequested.emit();
```

Comment:
This is poping the confirmation all the time but because is also deleting the box right away you don't see it, in slower devices users might me able to see it and it will provide a bad experience.

What about if instead of this we pass a input to show or not the confirmation?

Reply:
@fmontes good idea! Will change the implementation with that.